vtkext/private/module/vtkF3DUIActor.cxx

```diff
@@ -61,9 +67,13 @@ void vtkF3DUIActor::SetFpsValue(int fps)
 }
 
 //----------------------------------------------------------------------------
-void vtkF3DUIActor::SetFontFile(const std::string& font)
+void vtkF3DUIActor::SetFontFile(vtkOpenGLRenderWindow* renWin, const std::string& font)
 {
-  this->FontFile = font;
+  if (this->FontFile != font)
+  {
+    this->FontFile = font;
+    this->Initialize(renWin);
```

Comment:
not a fan of calling initialization code in a setter. Can you set a flag instead and initialize when you need it if the flag is set ?